pkg/hctltest/testdata/heating_climate_set_temperature_response.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I know that there's sometimes empty responses from the Home Assistant API. This is also the reason, why `hctl` doesn't really check the response at the moment. Do you really always get an empty response or could you maybe provide an example response for when the API is responding like expected?

Reply:
I'm always getting an empty response when calling `climate.set_temperature` - they don't seem to return the entity's state at all.

Reply:
ok, thanks for checking